contents/code/tile.js

```diff
@@ -202,12 +202,6 @@ Tile.prototype.setClientGeometry = function(client) {
             print("Wrong tile called");
             return;
         }
-        // Don't resize when we aren't the current desktop
-        // or on all desktops
-        if (this._currentDesktop != workspace.currentDesktop
-            && this._currentDesktop != -1) {
-            return;
-        }
```

Comment:
What's the rationale for removing this?

It should still work. The idea is that we restore the geometry when we switch desktops, so changes to a desktop while it's in the background don't affect the tiling and we can ignore a bunch of work.

Or is there a problem with that?

Reply:
Yep, as I wrote in the commit message, Desktop Grid effect displays damaged layouts if we don't keep them maintained all the time.

Reply:
Urgh, yeah, sorry, missed that!